rails-snow-queen/app/views/quotes/new.html.erb

```diff
@@ -1,17 +1,25 @@
-<h1>Quote calculator</h1>
+<div class="d-flex row">
+   <div id="panel" class="col-4 p-5">
+      <h1>Quote calculator</h1>
+      <p>Get a free quote for a salting or snow clearing service</p>
 
-<p>Get a free quote for a salting or snow clearing service</p>
+      <form action="" id="AddressSearch">
+         <label for="address">Search your location:</label><br>
+         <input id="address" type="textbox" placeholder="123 Street Rd, Port Moody" size="40">
```

Comment:
`textbox` is not a valid type, is it? I think you mean `text`

Reply:
should I actually make this a form_for or form_tag?

Reply:
If you're going to eventually Reactify it, don't bother, it's going to get in your way. But what you will need is to add `name` attributes to your fields. But perhaps you want to leave that for when you actually submit the form (other branch?)